hashtree_abi.nimble

```diff
@@ -0,0 +1,11 @@
+mode = ScriptMode.Verbose
+
+packageName   = "hashtree_abi"
+version       = "0.1.0"
```

Comment:
Note to Prysmatic: Nimble packages are released by tagging the repo with `v0.1.0` in this case. That's what gets downloaded when `nimble install` is requested for `0.1.0`. Right now, there don't seem to be any releases yet, but there is a tiny maintenance burden of keeping the latest tag in sync with what's written here.